### DISCUSSION TITLE
Raspberrymatic and old wired Homematic devices

### DISCUSSION THREAD

**Discussion by isharky** [OpenCCU/OpenCCU#2436] [🇩🇪 Fragen&Antworten]
Hi there,
 
I have old Homematic Wired devices installed im my house with a Homematic RS485 Lan Gateway. 
I have currently my home automation running on FHEM but I would like to switch to Home Assistant which is much better.

I have now a RPI4 with Home Assistant installed and Raspberrymatic installed as an addon. 
Should I install any additional device on my RPI4 to have the Raspberrymatic communicate with the RS485 Lan Gateway? (I do not have HM wireless devices, only wired)

Thanks for your help.

**Comment by jens-maus** [comment]
Well, this is a good question which I can personally not completely answer since I do not have any of these old RS485 LAN Gateways for BidCos-Wired. However, in my understand this should exactly work like you hope. Thus, all you need to do is to bind your existing RS485 LAN Gateway to the new RaspberryMatic system (no matter if this is an HA addon driven installation or a real install on a dedicated Pi). That means, switch of FHEM and then register that LAN Gateway to the new RaspberryMatic. Then reboot RaspberryMatic and see if the Teaching mode for BidCos-Wired appears in the teaching webui. And if this is the case you might teach-in all your BidCos-Wired devices one after another. However, it might be that you would have to actually factory reset all these devices which would of course also result in loosing your direct connection settings and device settings, which you should write down before you do so.

But in principle, just binding a RS485 BidCos-Wired LAN gateway to RaspberryMatic should work without having to install/use one of the RF gateway solutions like HM-MOD-RPI-PCB or RPI-RF-MOD or HmIP-RFUSB.
